web/template/certificate_provider_who_is_eligible.gohtml

```diff
@@ -0,0 +1,40 @@
+{{ template "page" . }}
+
+{{ define "pageTitle" }}{{ tr .App "whoIsEligibleToBeCertificateProvider" }}{{ end }}
+
+{{ define "main" }}
+    <div class="govuk-notification-banner govuk-notification-banner--success" role="alert"
+         aria-labelledby="govuk-notification-banner-title"
+         data-module="govuk-notification-banner">
+        <div class="govuk-notification-banner__header">
+            <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">
+                {{ tr .App "success" }}
```

Comment:
having a success notification here won't make sense until we move entering the reference number to after signing in

Reply:
TEchnically true, although from the users perspective they have just entered the ref number so this should still work when we shift the order around.